docs/UserGuide.md

```diff
@@ -429,27 +466,28 @@ ________________________________________________________________________________
   exit        | To exit DietBook:
               | exit
 __________________________________________________________________________________________________________________________________________
-````
- #### To exit DietBook: `exit`
- 
- Exits the DietBook.
+```
+
+#### To exit DietBook: `exit`
+
+Exits the DietBook.
  
- Format: `exit`
+Format: `exit`
  
- Output example:
- ```
- Bye! Hope to see you again soon!
- ```
+Output example:
+```
+Bye! Hope to see you again soon!
```

Comment:
Is this aiming to fix the error I sent you?

Reply:
I believe it should. However, I am unable to confirm without merging and checking the website.